can-stache-bindings.js

```diff
@@ -265,13 +269,17 @@ var behaviors = {
 			if(domData.get.call(el, "preventDataBindings")) {
 				return;
 			}
-			var viewModel = canViewModel(el),
+			var viewModel,
+				getViewModel = function() {
+					return viewModel || (viewModel = canViewModel(el));
+				},
```

Comment:
Because otherwise a viewModel will always be set up on the element and we will always use `vm` binding instead of element binding.